--- bng2/Perl2/SpeciesGraph.pm
+++ bng2/Perl2/SpeciesGraph.pm
@@ -819,6 +819,7 @@
	my $err = '';  # return error (set string if species compartment is invalid)

	my @missing_comp_mols = ();
+	my @missing_comp_components = ();

	# Gather molecule compartments and component compartments
	my %mol_comps = ();
@@ -847,6 +848,16 @@
		foreach my $component ( @{ $mol->Components } )
		{
			my $ccomp = $component->Compartment;
+
+			# Fallback to molecule's compartment if defined and component's is not
+			if ( !(defined $ccomp) and defined $comp )
+			{
+				$ccomp = $comp;
+			}
+
+			if ( !(defined $ccomp) )
+			{
+				push @missing_comp_components, $component;
+			}
+
			if ( defined $ccomp )
			{
				if    ( $ccomp->SpatialDimensions == 2 ) { $surfaces{$ccomp} = $ccomp; }
@@ -863,6 +874,12 @@
		$err = sprintf "Molecule %s in SpeciesGraph %s does not have a specified compartment.", $missing_comp_mols[0]->Name, $sg->toString();
		return ( undef, $err );
	}
+
+	if ( @missing_comp_components > 0 && ($n_surfaces > 0 || $n_volumes > 0 || defined $sg->Compartment) )
+	{
+		$err = sprintf "Component %s in SpeciesGraph %s does not have a specified compartment.", $missing_comp_components[0]->Name, $sg->toString();
+		return ( undef, $err );
+	}

	# infer Species Compartment
	if ( $n_surfaces == 0 )
