---
title: Applications
layout: default
permalink: /applications
parent: Uses of BioNetGen
nav_order: 1
---

# Published BioNetGen Models

This page links to publications that have applied a BioNetGen rule-based model to a specific biological system. *Note: This list is a work in progress and may not be complete.*

Help keep this page current by sending your publications to our [email](mailto:bionetgen.help@gmail.com) or [Twitter](http://twitter.com/bionetgen).

## Preprints
* Korwek Z et al. [Non-self RNA rewires IFNβ signaling: A mathematical model of the innate immune response](https://www.biorxiv.org/content/10.1101/2022.01.30.478391v1.full) *bioRxiv*

## 2021
* McMillan D et al. [Structural insights into the disruption of TNF-TNFR1 signalling by small molecules stabilising a distorted TNF.](https://pubmed.ncbi.nlm.nih.gov/33495441/) *Nat. Commun.*
* Zhang Y et al. [A systems biology model of junctional localization and downstream signaling of the Ang-Tie signaling pathway.](https://pubmed.ncbi.nlm.nih.gov/34417472/) *NPJ Syst. Biol. Appl.*
* Erdem C et al. [Inhibition of RPS6K reveals context-dependent Akt activity in luminal breast cancer cells.](https://pubmed.ncbi.nlm.nih.gov/34191793/) *PLOS Comput. Biol.*

## 2020
* Bolado-Carrancio A et al. [Periodic propagating waves coordinate RhoGTPase network dynamics at the leading and trailing edges during cell migration.](https://pubmed.ncbi.nlm.nih.gov/32705984/) *Elife*
* Ordyan M et al. [Interactions between calmodulin and neurogranin govern the dynamics of CaMKII as a leaky integrator.](https://pubmed.ncbi.nlm.nih.gov/32678848/) *PLOS Comput. Biol.*
* Kirsch K et al. [Co-regulation of the transcription controlling ATF2 phosphoswitch by JNK and p38.](https://pubmed.ncbi.nlm.nih.gov/33188182/) *Nat. Commun.*
* Salzer B et al. [Engineering AvidCARs for combinatorial antigen recognition and reversible control of CAR function.](https://pubmed.ncbi.nlm.nih.gov/32820173/) *Nat. Commun.*
* Wu Q, Finley SD. [Mathematical Model Predicts Effective Strategies to Inhibit VEGF-eNOS Signaling.](https://pubmed.ncbi.nlm.nih.gov/32357492/) *J. Clin. Med.*
* Kapralov AA et al. [Redox lipid reprogramming commands susceptibility of macrophages and microglia to ferroptotic death.](https://pubmed.ncbi.nlm.nih.gov/32080625/) *Nat. Chem. Biol.*
* Paoletti F et al. [Molecular flexibility of DNA as a key determinant of RAD51 recruitment.](https://pubmed.ncbi.nlm.nih.gov/31943278/) *EMBO J.*
* Salazar-Cavazos E et al. [Multisite EGFR phosphorylation is regulated by adaptor protein abundances and dimer lifetimes.](https://pubmed.ncbi.nlm.nih.gov/31913761/) *Mol. Biol. Cell*

## 2019
* Erickson KE et al. [Modeling cell line-specific recruitment of signaling proteins to the insulin-like growth factor 1 receptor.](https://pubmed.ncbi.nlm.nih.gov/30653502/) *PLOS Comput. Biol.*
* Nikolaev Y et al. [Systems NMR: single-sample quantification of RNA, proteins and metabolites for biomolecular network analysis.](https://pubmed.ncbi.nlm.nih.gov/31363225/) *Nat. Methods*
* Lin YT, Feng S, Hlavacek WS. [Scaling methods for accelerating kinetic Monte Carlo simulations of chemical reaction networks.](https://pubmed.ncbi.nlm.nih.gov/31255063/) *J. Chem. Phys.*

## 2018 
* Wong VC et al. [NF-κB-Chromatin Interactions Drive Diverse Phenotypes by Modulating Transcriptional Noise.](http://www.ncbi.nlm.nih.gov/pubmed/29346759) *Cell Reports*
* Bazzazi J et al.[Computational modeling of synergistic interaction between αVβ3 integrin and VEGFR2 in endothelial cells: Implications for the mechanism of action of angiogenesis-modulating integrin-binding peptides.](https://www.ncbi.nlm.nih.gov/pubmed/30036530) *J. Theor. Biol.*
* Tse MJ et al. [Rare-event sampling of epigenetic landscapes and phenotype transitions.](https://pubmed.ncbi.nlm.nih.gov/30074987/) *PLOS Comput. Biol.*
* Singh M et al. [Shift from stochastic to spatially-ordered expression of serine-glycine synthesis enzymes in 3D microtumors.](https://pubmed.ncbi.nlm.nih.gov/29925909/) *Sci. Rep.*
* Rohrs JA et al. [Computational Model of Chimeric Antigen Receptors Explains Site-Specific Phosphorylation Kinetics.](https://pubmed.ncbi.nlm.nih.gov/30197180/) *Biophys. J.*
* James JR. [Tuning ITAM multiplicity on T cell receptors can control potency and selectivity to ligand density.](https://pubmed.ncbi.nlm.nih.gov/29789296/) *Sci. Signal*
* Czerkies M et al. [Cell fate in antiviral response arises in the crosstalk of IRF, NF-κB and JAK/STAT pathways.](https://pubmed.ncbi.nlm.nih.gov/29402958/) *Nat. Commun.*
* Rukhlenko OS et al. [Dissecting RAF Inhibitor Resistance by Structure-based Modeling Reveals Ways to Overcome Oncogenic RAS Signaling](https://pubmed.ncbi.nlm.nih.gov/30007540/) *Cell Syst.*

## 2017
* Harmon B et al. [Timescale Separation of Positive and Negative Signaling Creates History-Dependent Responses to IgE Receptor Stimulation.](https://pubmed.ncbi.nlm.nih.gov/29138425/) *Sci. Rep.*
* Meng X et al. [Minimum-noise production of translation factor eIF4G maps to a mechanistically determined optimal rate control window for protein synthesis.](https://pubmed.ncbi.nlm.nih.gov/27928055/) *Nucleic Acids Res.*
* Bazzazi J, Isenberg JS, Popel AS. [Inhibition of VEGFR2 Activation and Its Downstream Signaling to ERK1/2 and Calcium by Thrombospondin-1 (TSP1): *In silico* Investigation.](https://pubmed.ncbi.nlm.nih.gov/28220078/) *Front Physiol.*
* Bazzazi J, Popel AS. [Computational investigation of sphingosine kinase 1 (SphK1) and calcium dependent ERK1/2 activation downstream of VEGFR2 in endothelial cells.](https://pubmed.ncbi.nlm.nih.gov/28220078/) *PLoS Comput. Biol.*

## 2016
* Antunes G, Roque AC, Simoes de Souza FM. [Modelling intracellular competition for calcium: kinetic and thermodynamic control of different molecular modes of signal decoding.](http://www.ncbi.nlm.nih.gov/pubmed/27033299) *Sci. Rep.*
* Hat B et al. [Feedbacks, Bifurcations, and Cell Fate Decision-Making in the p53 System.](http://www.ncbi.nlm.nih.gov/pubmed/26928575) *PLOS Comput. Biol.*
* Rohrs JA, Sulistio CD, Finley SD. [Predictive model of thrombospondin-1 and vascular endothelial growth factor in breast tumor tissue.](https://pubmed.ncbi.nlm.nih.gov/28713587/) *NPJ Syst. Biol. Appl.*
* Korwek Z et al. [Importins promote high-frequency NF-κB oscillations increasing information channel capacity.](https://pubmed.ncbi.nlm.nih.gov/27835978/) *Biol. Direct*
* Camillo BD et al. [A rule-based model of insulin signalling pathway.](https://pubmed.ncbi.nlm.nih.gov/27245161/) *BMC Syst. Biol.*

## 2015
* Dolan DWP et al. [Integrated Stochastic Model of DNA Damage Repair by Non-homologous End Joining and p53/p21- Mediated Early Senescence Signalling.](http://www.ncbi.nlm.nih.gov/pubmed/26020242) *PLOS Comput. Biol.*
* Stites EC et al. [Use of mechanistic models to integrate and analyze multiple proteomic datasets.](http://www.ncbi.nlm.nih.gov/pubmed/25863072) *Biophys. J.*
* Hawse WF et al. [Cutting Edge: Differential Regulation of PTEN by TCR, Akt, and FoxO1 Controls CD4+ T Cell Fate Decisions.](http://www.ncbi.nlm.nih.gov/pubmed/25855357) *J. Immunol.*
* Szymańska P et al. [Computational analysis of an autophagy/translation switch based on mutual inhibition of MTORC1 and ULK1.](http://www.ncbi.nlm.nih.gov/pubmed/25761126) *PLOS One*
* Birtwistle MR. [Analytical reduction of combinatorial complexity arising from multiple protein modification sites.](http://www.ncbi.nlm.nih.gov/pubmed/25519995) *J. R. Soc. Interface*

## 2014
* Ligon TS, Leonhardt C, Rädler JO. [Multi-level kinetic model of mRNA delivery via transfection of lipoplexes.](http://www.ncbi.nlm.nih.gov/pubmed/25237886) *PLOS One*
* Chylek LA et al. [Phosphorylation site dynamics of early T-cell receptor signaling.](http://www.ncbi.nlm.nih.gov/pubmed/25147952) *PLOS One*
* Dushek O et al. [Biosensor architectures for high-fidelity reporting of cellular signaling.](http://www.ncbi.nlm.nih.gov/pubmed/25099816) *Biophys. J.*
* Chylek LA et al. [An Interaction Library for the FcεRI Signaling Network.](http://www.ncbi.nlm.nih.gov/pubmed/24782869) *Frontiers Immunol.*
* Kozer N et al. [Recruitment of the adaptor protein Grb2 to EGFR tetramers.](http://www.ncbi.nlm.nih.gov/pubmed/24697349) *Biochemistry*

## 2013 
* Ibrahim B et al. [Spatial rule-based modeling: a method and its application to the human mitotic kinetochore.](http://www.ncbi.nlm.nih.gov/pubmed/24709796) *Cells*
* Pękalski J et al. [Spontaneous NF-κB activation by autocrine TNFα signaling: a computational analysis.](http://www.ncbi.nlm.nih.gov/pubmed/24324544) *PLOS One*
* Falkenberg CV, Blinov MJ, Loew LM. [Pleomorphic ensembles: formation of large clusters composed of weakly interacting multivalent molecules.](http://www.ncbi.nlm.nih.gov/pubmed/24314076) *Biophys. J.*
* Barua D, Hlavacek WS. [Modeling the effect of APC truncation on destruction complex function in colorectal cancer cells.](http://www.ncbi.nlm.nih.gov/pubmed/24086117) *PLOS Comput. Biol.*
* Fengos G, Iber D. [Prediction stability in a data-based, mechanistic model of σF regulation during sporulation in Bacillus subtilis.](http://www.ncbi.nlm.nih.gov/pubmed/24067622) *Sci. Rep.*
* Kozer N et al. [Exploring higher-order EGFR oligomerisation and phosphorylation--a combined experimental and theoretical approach.](http://www.ncbi.nlm.nih.gov/pubmed/23629589) *Mol. Biosyst.*
* Chow SK et al. [Disease-enhancing antibodies improve the efficacy of bacterial toxin-neutralizing antibodies.](http://www.ncbi.nlm.nih.gov/pubmed/23601104) *Cell Host Microbe*
* Mukhopadhyay H et al. [Systems model of T cell receptor proximal signaling reveals emergent ultrasensitivity.](http://www.ncbi.nlm.nih.gov/pubmed/23555234) *PLOS Comput. Biol.*
* Liu Y et al. [Single-cell measurements of IgE-mediated FcεRI signaling using an integrated microfluidic platform.](http://www.ncbi.nlm.nih.gov/pubmed/23544131) *PLOS One*
* Falkenberg CV, Loew LM.[Computational analysis of Rho GTPase cycling.](http://www.ncbi.nlm.nih.gov/pubmed/23326220) *PLOS Comput. Biol.*
* Kesseler KJ et al. [A predictive mathematical model of the DNA damage G2 checkpoint.](http://www.ncbi.nlm.nih.gov/pubmed/23266715) *J. Theor. Biol.*

## 2012
* Barua D, Goldstein B. [A mechanistic model of early FcεRI signaling: lipid rafts and the question of protection from dephosphorylation.](http://www.ncbi.nlm.nih.gov/pubmed/23284735) *PLOS One*
* Creamer MS et al. [Specification, annotation, visualization and simulation of a large rule-based model for ERBB receptor signaling.](http://www.ncbi.nlm.nih.gov/pubmed/22913808) *BMC Syst. Biol.*
* Barua D, Hlavacek WS, Lipniacki T. [A computational model for early events in B cell antigen receptor signaling: analysis of the roles of Lyn and Fyn.](http://www.ncbi.nlm.nih.gov/pubmed/22711887) *J. Immunol.*
* Michalski PJ, Loew LM. [CaMKII activation and dynamics are independent of the holoenzyme structure: an infinite subunit holoenzyme approximation.](http://www.ncbi.nlm.nih.gov/pubmed/22683827) *Phys. Biol.*
* Kocieniewski P, Faeder JR, Lipniacki T.[The interplay of double phosphorylation and scaffolding in MAPK pathways.](http://www.ncbi.nlm.nih.gov/pubmed/22123371) *J. Theor. Biol.*

## 2011
* Geier F, Fengos G, Iber D. [A computational analysis of the dynamic roles of talin, Dok1, and PIPKI for integrin activation.](http://www.ncbi.nlm.nih.gov/pubmed/22110576) *PLOS One*
* Dushek O, van der Merwe PA, Shahrezaei V. [Ultrasensitivity in multisite phosphorylation of membrane-anchored proteins.](http://www.ncbi.nlm.nih.gov/pubmed/21354391) *Biophys. J.*
* Thomson TM et al. [Scaffold number in yeast signaling system sets tradeoff between system output and dynamic range.](http://www.ncbi.nlm.nih.gov/pubmed/22114196) *PNAS*

## 2010
* Nag A, Faeder JR, Goldstein B. [Shaping the response: the role of FcεRI and Syk expression levels in mast cell signaling.](http://www.ncbi.nlm.nih.gov/pubmed/21073233) *IET Syst. Biol.*
* Nag A et al. [A detailed mathematical model predicts that serial engagement of IgE-FcεRI complexes can enhance Syk activation in mast cells.](http://www.ncbi.nlm.nih.gov/pubmed/20733205) *J. Immunol.*
* Monine MI et al. [Modeling multivalent ligand-receptor interactions with steric constraints on configurations of cell-surface receptor aggregates.](http://www.ncbi.nlm.nih.gov/pubmed/20085718) *Biophys. J.*
* Gong J et al. [Analysis and verification of the HMGB1 signaling pathway.](http://www.ncbi.nlm.nih.gov/pubmed/21106117) *BMC Bioinformatics*
* Ray JC, Igoshin OA. [Adaptable functionality of transcriptional feedback in bacterial two-component systems.](http://www.ncbi.nlm.nih.gov/pubmed/20168997) *PLOS Comput. Biol.*

## 2009
* Dushek O, Das R, Coombs D. [A role for rebinding in rapid and reliable T cell responses to antigen.](http://www.ncbi.nlm.nih.gov/pubmed/19956745) *PLOS Comput. Biol.*
* Barua D, Faeder JR, Haugh JM. [A bipolar clamp mechanism for activation of Jak-family protein tyrosine kinases.](http://www.ncbi.nlm.nih.gov/pubmed/19381268) *PLOS Comput. Biol.*
* An GC, Faeder JR. [Detailed qualitative dynamic knowledge representation using a BioNetGen model of TLR-4 signaling and preconditioning.](http://www.ncbi.nlm.nih.gov/pubmed/18835283) *Math. Biosci.*
* Nag A et al. [Aggregation of membrane proteins by cytosolic cross-linkers: theory and simulation of the LAT-Grb2-SOS1 system.](http://www.ncbi.nlm.nih.gov/pubmed/19348745) *Biophys. J.*

## 2008
* Barua D, Faeder JR, Haugh JM. [Computational models of tandem SRC homology 2 domain interactions and application to phosphoinositide 3-kinase.](http://www.ncbi.nlm.nih.gov/pubmed/18204097) *J. Biol. Chem.*

## 2007
* Mu F et al. [Carbon-fate maps for metabolic reactions.](http://www.ncbi.nlm.nih.gov/pubmed/17933853) *Bioinformatics*
* Barua D, Faeder JR, Haugh JM. [Structure-based kinetic models of modular signaling protein function: focus on Shp2.](https://pubmed.ncbi.nlm.nih.gov/17208977/) *Biophys. J.*

## 2006
* Blinov ML et al. [A network model of early events in epidermal growth factor receptor signaling that accounts for combinatorial complexity.](http://www.ncbi.nlm.nih.gov/pubmed/16233948) *Biosystems*

## 2005
* Faeder JR et al. [Combinatorial complexity and dynamical restriction of network flows in signal transduction.](http://www.ncbi.nlm.nih.gov/pubmed/17091578) *IET Syst. Biol.*

## 2003 
* Faeder JR et al. [Investigation of early events in FcεRI-mediated signaling using a detailed mathematical model.](http://www.ncbi.nlm.nih.gov/pubmed/12646643) *J. Immunol.*
